examples/des/blend.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2012-2015 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+
+
+# A simulation of blended pairs of galaxies at various separations and flux ratios.
+# There are two parallel runs, one normal and the other where the neighbor only contributes
+# to the noise, not the flux.  So this simulates a perfect deblender that removes the flux
+# correctly, but there is still the extra noise from the neighbor.
```

Comment:
I instead made the ngmix dependency optional.  If ngmix is there, it uses it, otherwise, it has its own implementation.  961883a60b58cec9a311c35481e634be622d785c

I also fixed a few things that got messed up in the merge from #693.  So I think all the scripts in the des directory should be runnable now.